.travis.yml

```diff
@@ -32,20 +32,25 @@ before_install:
 install:
   - conda install --yes pip
   - conda install --yes --file requirements.txt;
-  - pip install https://github.com/pysal/splot/archive/master.zip
+  - git clone https://github.com/pysal/splot.git;
```

Comment:
Not sure why `splot` is installed here for testing?

Reply:
I thought that was a bit out of place, as well. Shall I see if this PR passes without `splot`?

Reply:
travis passes without `splot`. not sure how that was in there to begin with.